realm/realm-library/src/syncIntegrationTest/java/io/realm/SyncSessionTests.java

```diff
@@ -652,7 +642,9 @@ public void stop_multipleTimes() {
 
     @Test
     @RunTestInLooperThread
+    @Ignore("FIXME https://github.com/realm/realm-sync/issues/3234")
```

Comment:
The Java & Core issue related to this seems closed, do you want to enable this again?

Reply:
Yup, reenabling.